### PR TITLE
chore(changelog): update changelog for v4.1.0

### PR DESCRIPTION
**Pull request by joshua-goldstein**
Update changelog for v4.1.0.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,30 @@ All notable changes to this project will be documented in this file.
...
+
+### Features
+- feat(stream): add support for incremental stream writer (#1722) (#1874)
+
+### Chores
+
+- fix(docs): update README (#1915)
+- cleanup sstable file after tests (#1912)
+- chore(ci): add dgraph regression tests (#1908)
+- chore: upgrade xxhash from v1.1.0 to v2.1.2 (#1910)
```
Comment (mangalaman93) [inline CHANGELOG.md:19]: This actually fixes a bug

**Review by mangalaman93 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (joshua-goldstein) [reply CHANGELOG.md]: mentioned bug

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,30 @@ All notable changes to this project will be documented in this file.
...
+- docs: fix the default value in docs (#1909)
+- chore: update URL for unsupported manifest version error (#1905)
+- docs(README): add raft-badger to projects using badger (#1902)
+- sync the docs with README with projects using badger (#1903)
+- fix: update code comments for WithNumCompactors (#1900)
+- chore(deps): bump golang.org/x/net from 0.0.0-20201021035429-f5854403a974 to 0.7.0 (#1885)
```
[the remark this answers is not in the document]
Comment (joshua-goldstein) [reply CHANGELOG.md]: added cve's

**Review by mangalaman93 — APPROVED** [review]

**Review by mangalaman93 — APPROVED** [review]

**Review by skrdgraph — APPROVED** [review]
👍

**Pull request merged**